pywinauto/__init__.py

```diff
@@ -163,3 +163,8 @@ def top_from_point(self, x, y):
         """Get wrapper object for top level element at specified screen coordinates (x, y)"""
         top_element_info = self.backend.element_info_class.top_from_point(x, y)
         return self.backend.generic_wrapper_class(top_element_info)
+
+    def from_focused(self):
+        """Get wrapper object for element with focus"""
+        element_info = self.backend.element_info_class.get_active()
+        return self.backend.generic_wrapper_class(element_info)
```

Comment:
Need empty line at end of file.

Reply:
Hmm something my git settings must be stripping out the empty lines.

Reply:
done